SAMtags.tex

```diff
@@ -96,6 +97,7 @@ \section{Standard tags}
   {\tt PU} & Z & Platform unit \\
   {\tt Q2} & Z & Phred quality of the mate/next segment sequence in the {\tt R2} tag \\
   {\tt QT} & Z & Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag \\
+  {\tt CY} & Z & Phred quality of the cellular-barcode sequence in the {\tt CR} tag \\
```

Comment:
Can you move this after `CT` so it's alphabetical?

Reply:
Oh boy, the other tags are not alphabetically ordered.  Perhaps a second commit to fix that while your changing things?

Reply:
FYI the intention in the SAMtags document was for the §1 table to be alphabetically ordered, but for the individual subsections' tags to be ordered in some semantically meaningful way for that particular subsection. For example, in §1.2 (Metadata) RG is fundamental while CO is trivial, and the subsection is ordered accordingly. YMMV as to what is semantically meaningful for a barcodes subsection.

Reply:
OK, I resorted the tags in the Barcodes section in a semantically meaningful way.